sql/driver/driver_test.go

```diff
@@ -48,9 +49,10 @@ func TestPlaceholders(t *testing.T) {
 	s, db := setup(t)
 	defer cleanup(s, db)
 
-	year, month, day := 3015, time.August, 30
-	timeVal := time.Date(year, month, day, 3, 34, 45, 345670000, time.UTC)
-	dateVal := time.Date(year, month, day, 0, 0, 0, 0, time.UTC)
+	timeVal := time.Date(3015, time.August, 30, 3, 34, 45, 345670000, time.UTC)
+	// date must be in local time because the server will return a date
+	// in local time.
```

Comment:
The timezone itself doesn't matter, but because the new truncation logic is now correctly timezone aware (see other comment), the date's time zone does matter (because it affects the date to which it will be truncated).